test/functional/fixtures/api/es-next/selector/testcafe-fixtures/selector-test.js

```diff
@@ -1085,3 +1085,10 @@ test('Selector `addCustomMethods` method - Selector mode', async t => {
     await t.expect(await nonExistingElement()).eql(null);
 });
 
+test('Inexistent element with exists/visible checks', async t => {
```

Comment:
It won't be run until you also add a Mocha test that references this TestCafe test here: https://github.com/DevExpress/testcafe/blob/cfa8d81a3ef8c4a8e33d6887394c3decb00115e0/test/functional/fixtures/api/es-next/selector/test.js